app/__init__.py

```diff
@@ -48,12 +48,48 @@ def create_app(config_class, database_uri=None):
         ]
     )
 
-    # Set content security policy
+    SELF = "'self'"
+
     csp = {
-        "default-src": f"'self' {app.config['FLASKS3_CDN_DOMAIN']}",
-        "script-src": f"'self' {app.config['FLASKS3_CDN_DOMAIN']}"
-        + " 'sha256-GUQ5ad8JK5KmEWmROf3LZd9ge94daqNvd8xy9YS1iDw='"  # pragma: allowlist secret
-        + " 'sha256-l1eTVSK8DTnK8+yloud7wZUqFrI0atVo6VlC6PJvYaQ='",  # pragma: allowlist secret
+        "default-src": f"'self' {app.config['FLASKS3_CDN_DOMAIN']} https://cdn.jsdelivr.net 'unsafe-inline' ",
+        "script-src": (
+            [
+                SELF,
+                "'unsafe-eval'",
+                f"{app.config['FLASKS3_CDN_DOMAIN']}",
+                "https://cdn.jsdelivr.net",
+                "https://cdnjs.cloudflare.com",
```

Comment:
is this allowing any scripts from these domains? Is it possible to lock it down to specific urls of those we need, or do they change dynamically?

Reply:
Reviewed on call and updated while pairing. Awaiting a review from AJ